documentation/Overview.md

```diff
@@ -4,24 +4,24 @@ permalink: /documentation/overview/
 title: Overview
 weight: 2
 ---
-# REACT Reference Storefront
+# React Reference Storefront
 
 ## Overview
 
-The REACT PWA Reference Storefront is a flexible e-commerce website built on Elastic Path’s [RESTful](http://en.wikipedia.org/wiki/Representational_state_transfer) e-commerce API,  [Cortex API](https://developers.elasticpath.com/commerce/7.3/Cortex-API-Front-End-Development/Getting-Started/Introduction).Through Cortex API, the storefront get to use the ecommerce capabilities provided Elastic Path Commerce and get data in a RESTful manner.
+The React PWA Reference Storefront is a flexible e-commerce website built on Elastic Path’s [RESTful](http://en.wikipedia.org/wiki/Representational_state_transfer) e-commerce API,  [Cortex API](https://developers.elasticpath.com/commerce/7.3/Cortex-API-Front-End-Development/Getting-Started/Introduction). Through Cortex API, the storefront get to use the e-commerce capabilities provided by Elastic Path Commerce and get data in a RESTful manner.
 
-React Reference Storefront is divided into containers(Pages) and React components. Each page in the storefront is composed of various components that fulfills the purpose of the page. For example, home, cart, categories, checkout, and registration are separate Pages. Each page consists of various components, such as navigation, footer, product List, or products.
+React Reference Storefront is composed of containers (pages) and React components. Each page in the storefront is composed of various components that fulfills the purpose of the page. For example, home, cart, categories, checkout, and registration are separate pages. Each page consists of various components, such as navigation, footer, product List, or products.
```

Comment:
In the last sentence, the "List" is capitalized, but none of the other things are.

Reply:
Good catch